### PR TITLE
Fixing wso2event publisher sample binary  data receiver

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

File: modules/samples/consumers/wso2-event/src/main/java/org/wso2/carbon/sample/server/TestWso2EventServer.java [hunk 1]
```diff
@@ -119,10 +120,16 @@ public void receive(List<Event> eventList, Credentials credentials) {
 
         if (protocol.equalsIgnoreCase("binary")) {
             binaryDataReceiver = new BinaryDataReceiver(new BinaryDataReceiverConfiguration(receiverPort + 100, receiverPort), databridge);
+            try {
+                binaryDataReceiver.start();
+            }
```
Comment (ramindu90) [inline modules/samples/consumers/wso2-event/src/main/java/org/wso2/carbon/sample/server/TestWso2EventServer.java]: minor formatting issues. Please fix.

Comment (thiliA) [inline modules/samples/consumers/wso2-event/src/main/java/org/wso2/carbon/sample/server/TestWso2EventServer.java]: @ramindu90 This has been updated with a later fix.

**Comment by thiliA** [comment]
@ramindu90 
Thank you for pointing out. Fixed in PR: https://github.com/wso2/product-cep/pull/186

File: modules/samples/consumers/wso2-event/src/main/java/org/wso2/carbon/sample/server/TestWso2EventServer.java (hunk 1 above)
Comment (thiliA) [inline modules/samples/consumers/wso2-event/src/main/java/org/wso2/carbon/sample/server/TestWso2EventServer.java]: Thanks. Fixed in commit: https://github.com/thiliA/product-cep/commit/c9313f04e73aba5750c51ac8600a93216d264ccb

**Pull request merged**